apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -402,6 +402,11 @@ type OrchestratorExplorerFeatureConfig struct {
 	// +optional
 	ScrubContainers *bool `json:"scrubContainers,omitempty"`
 
+	// +optional
+	// `CustomResourcesCollection` defines custom resources for the orchestrator explorer to collect.
+	// Each item should follow the convention `group/version/kind`. For example, `datadoghq.com/v1alpha1/datadogmetrics`.
+	CustomResourcesCollection []string `json:"customResourcesCollection,omitempty"`
```

Comment:
i wonder if we could rename this to `CustomResources` since it is a list of custom resources, similar to how `ExtraTags` is a list of extra tags? from this current name i initially thought it would hold a bool